### ISSUE TITLE
Unable to reassign entries while deleting member

### ISSUE THREAD

**Issue by jelle-burodeng** [ExpressionEngine/ExpressionEngine#3157]
I have a site with 3 members. I need to delete 1 of them. I've set all members to show in the author list.
When i select the member ad hit delete, the modal shows "No members found"

PHP Version 8.1.15
EE 7.2.11

**Comment by intoeetive** [comment]
@jelle-burodeng can you add the screenshot of the error that you're seeing?

**Comment by jelle-burodeng** [comment]
![screenshot-error-reassign-entries](https://user-images.githubusercontent.com/107609890/229117929-8c875eda-d2e4-4739-9da3-16b956858b10.jpg)

**Comment by intoeetive** [comment]
And the other members that are authors, are those marked as authors individually - or as part of role that they belong to? Also, are you by chance trying to delete the only superadmin here?

**Comment by jelle-burodeng** [comment]
OK. found something that might clarify
This was a former super admin, whose roles we changed to "banned". 
When we set the roles back as Super Admin we did get the correct modal with an author list.
So now i changed him to "member" and the modal is empty again. Same when i give him a custom role. It looks like i can only reassign entries when that member has a role that is the same as the original role (SA in this case)
